proto/hummock.proto

```diff
@@ -267,7 +267,9 @@ message CompactTask {
   bool gc_delete_keys = 7;
   TaskStatus task_status = 9;
   // compaction group the task belongs to
-  uint64 compaction_group_id = 12;
+  uint64 compaction_group_id = 11;
+  // table ids in the compaction group when the task is generated
+  repeated uint32 group_table_ids = 12;
```

Comment:
Please fix this unnecessary breaking change.

Reply:
> Please fix this unnecessary breaking change.

Even if we do not break here, old compactors will set `group_table_ids` an empty vec and thus cannot report compact task.

Reply:
> > Please fix this unnecessary breaking change.
> 
> Even if we do not break here, old compactors will set `group_table_ids` an empty vec and thus cannot report compact task.

That is why I prefer the other alternative to not store id that doesn't belong to a state table stored in storage (i.e. sink id) in compaction group member table ids. In addition to not being a breaking change, this alternative also gives a clearer semantic for compaction group `member_table_id`. We shouldn't let compaction group aware of any non-storage concept like source/sink/actor/fragment id.